modules/claims_api/app/sidekiq/claims_api/service_base.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+
+module ClaimsApi
+  class ServiceBase
+    include Sidekiq::Job
+
+    RETRY_STATUS_CODES = %w[500 502 503 504].freeze
+
+    protected
+
+    def get_original_status_code(error)
+      if error.respond_to? :original_status
+        error.original_status
+      else
+        "No status code for error: #{error}"
```

Comment:
what's the use case for this and providing the `#{error}` and for that matter, any value? Is there something outside of the below? we're just checking if it's in the status code list above. If we were to be logging this or finding something to relay to make it a bit more meaningful I would be fine with keeping this but it seems kind of irrelevant as is? we're just passing this message for it to peruse the status codes and report false

Reply:
I would second updating the logging here to make it more meaningful in the scope of the DD monitoring work.  That way we can potentially monitor and alert if something comes back and meets this condition, ideally nothing ever would so if it does I think it would be useful to know.

Perhaps an update to utilize the claim logger and log tag inline with what is suggested below (or where ever that lands) with `claims_api_sidekiq_failure` and this text as the detail.